src/background.js

```diff
@@ -400,9 +400,11 @@ function main() {
             'utf8',
           )
 
+          const supportedVersion = "1.2"
```

Comment:
Create this constant in line 45
```suggestion
          const SUPPORTED_VERSION = '1.2'
```

Reply:
Called it `WITNET_RUST_VERSION` to be more explicit.